packages/seller-client/src/components/CopyableTextField/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './CopyableTextField';
```

Comment:
You're missing a line ending here. Make sure `eslint` is auto-formatting your work on save
```suggestion
export { default } from './CopyableTextField';

```